src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs

```diff
@@ -469,6 +469,11 @@
                 (range (inc (rand-int 5))))})
 
 
+(defn delete-agora-entity [type ns n sid]
+  {:path (str "/" type "s/" ns "/" n "/" sid)
```

Comment:
Rather than automatically pluralizing the type like this, I'd prefer the correct path segment to be passed in.